sherpa/optmethods/tests/_tstoptfct.cc

```diff
@@ -1344,7 +1348,8 @@ static PyObject *wood( PyObject *self, PyObject *args ) {
 
 static PyObject *init_optfcn( PyObject *self, PyObject *args ) {
 
-  int name_length, npar;
+  int npar;
+  Py_ssize_t name_length;
   char* name;
```

Comment:
Thanks for the comments. This is not my code, so I can't comment on its current set up. I will note that it's "test" code, so I think we are perhaps more forgiving here than we'd want to be in other parts of the code base ;-)

Reply:
> Side note: it is better make it `const`. It refers to immutable internal data in the Python string object.

Are you talking about making `name_length` or `npar` (or both) `const` here? If so, I don't see how to do that since we send them to `PyArg_ParseTuple` to be set. Or did you mean something else?